src/Microsoft.Health.Dicom.Web/Microsoft.Health.Dicom.Web.csproj

```diff
@@ -32,4 +32,10 @@
     <Folder Include="wwwroot\" />
   </ItemGroup>
 
+  <Target Name="SwaggerPostBuildTarget" AfterTargets="Build" Condition="'$(ContinuousIntegrationBuild)' != 'true'">
+    <Exec Command="dotnet tool restore"></Exec>
```

Comment:
Do we need to manually specify the manifest or will it find it automatically based on convention?